app/src/main/java/org/usfirst/team25/scouting/calculatorapp/MainActivity.java

```diff
@@ -37,7 +37,7 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
-        previousNumber = Double.parseDouble(getString(R.string.default_value));
+        previousNumber = Double.parseDouble("25");
```

Comment:
Why is the string resource replaced with a hardcoded value?

Reply:
Would this work instead?
`previousNumber = Double.parseDouble((String) mainDisplay.getText());`

Reply:
Yep, that should work as long as `mainDisplay.getText()` never returns an empty string